tagstudio/src/core/media_types.py

```diff
@@ -206,7 +219,11 @@ class MediaCategories:
         ".raw",
         ".rw2",
     }
-    _IMAGE_VECTOR_SET: set[str] = {".svg"}
+    _IMAGE_VECTOR_SET: set[str] = {
+        ".svg",
+        # ".eps",
+        # ".svgz",
```

Comment:
I haven't tested, but I believe this will break the current vector thumbnail renderer? I believe it is always assuming the file is an SVG currently. See: https://github.com/TagStudioDev/TagStudio/blob/main/tagstudio/src/qt/widgets/thumb_renderer.py#L783